### ISSUE TITLE
Suggestion: Auto Concede Players Who Leave The Game

### ISSUE THREAD

**Issue by SittingInTheFront** [jeremylarner/keyteki#143]
When players leave the game, auto concede them.

**Comment by chaoticserenitea** [comment]
👍🏻

**Comment by Grizzbear111** [comment]
Yes, and with a check if the player is the only left in the game. If players A and B in game, and either clicks Leave Game without disconnecting, cause game to auto-concede. When second player clicks leave game, since no players remain then no concede needed. I see more and more that leave when losing, just so they don't officially lose.
